test/verifier/universal-verifier.test.ts

```diff
@@ -389,17 +392,40 @@ describe("Universal Verifier tests", function () {
         .be.rejected;
     });
 
+    it("Check InvalidRequestOwner", async () => {
```

Comment:
Can this test be removed? It looks like it is already tested at Verifier contract level.